Source/interfaces/IPackager.h

```diff
@@ -57,13 +66,51 @@ namespace Exchange {
             enum { ID = ID_PACKAGER_NOTIFICATION };
             virtual void StateChange(IPackageInfo* package, IInstallationInfo* install) = 0;
             virtual void RepositorySynchronize(uint32_t status) = 0;
+            virtual void IntallStep(state status, uint32_t task, string id, int32_t code) = 0;
+        };
+
+        struct EXTERNAL IPackageInfoEx : virtual public Core::IUnknown {
+            enum { ID = ID_PACKAGER_PACKAGEINFOEX };
+
+            virtual ~IPackageInfoEx(){};
+
+            virtual string Name() const = 0;
+            virtual string BundlePath() const = 0;
+            virtual string Version() const = 0;
+            virtual string PkgId() const = 0;
+            virtual string Installed() const = 0;
+            virtual int64_t SizeInBytes() const = 0;
```

Comment:
Is there a meaning to negative Sizes in Bytes, or should this just be an unsigned (uint64_t) ?

Reply:
Changed to a more consistent **uint64_t**